ajax/libs/phery.js/package.json

```diff
@@ -0,0 +1,44 @@
+{  
+   "name":"phery.js",
+   "filename":"jquery.daterangepicker.min.js",
+   "description":"The bridge from jQuery to PHP, and vice-versa through AJAX.",
+   "license":"MIT",
+   "keywords":[  
+      "AJAX",
+      "event",
+      "unobtrusive",
+      "fileupload",
+      "PHP",
+      "form",
+      "serialization",
+      "partial",
+      "REST",
+      "upload",
+      "FormData",
+      "jquery",
+      "dom",
+      "HTML5",
+      "JSON"
+   ],
+   "repository":{  
+      "type":"git",
+      "url":"git://github.com/pheryjs/phery.gi"
+   },
+   "homepage":"http://phery-php-ajax.net",
+   "author":{  
+      "name":"Paulo Cesar",
+      "web":"http://github.com/pocesar"
```

Comment:
I think `"url"` will be the better key. The upstream use `"url"` for this. 
Ref: https://github.com/pheryjs/phery/blob/3.0.0-alpha2/phery.jquery.json#L7